src/index.ts

```diff
@@ -210,11 +222,21 @@ export default class EditorjsList {
     this.config = config;
     this.block = block;
 
+    /**
+     * Validate user configuration
+     */
+    validateConfig(this.config);
+
     /**
      * Set the default list style from the config or presetted 'unordered'.
      */
     this.defaultListStyle = this.config?.defaultStyle || 'unordered';
 
+    /**
+     * Set the style's list from the config
+     */
+    EditorjsList.styles = this.config?.styles;
```

Comment:
I'm not sure that setting to a static getter is a good idea. If you have several instances of Editor.js on the page with different configs, they can override each other.